core/store/migrations/migration1559767168/migrate.go

```diff
@@ -0,0 +1,29 @@
+package migration1559767168
+
+import (
+	"time"
+
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/store/assets"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"gopkg.in/guregu/null.v3"
+)
+
+func Migrate(tx *gorm.DB) error {
+	if err := tx.AutoMigrate(JobSpec{}).Error; err != nil {
+		return errors.Wrap(err, "failed to auto migrate JobSpec")
+	}
+	return nil
+}
+
+type JobSpec struct {
```

Comment:
Hmm, it looks like migration0 still points to the live JobSpec - that might cause problems for existing users. Have you tried migrating an old database?